### PR TITLE
Add tilegrid debug layer

### PR DESCRIPTION
**Pull request by vdubr**

### CODE REVIEW

File: src/ReactLeafletMap/layers/CanvasVectorLayer/genericCanvasLayer.js [hunk 1]
```diff
@@ -201,3 +201,5 @@ L.CanvasLayer = (L.Layer ? L.Layer : L.Class).extend({
 L.canvasLayer = function (options) {
 	return new L.CanvasLayer(options);
 };
+
+export default null;
```
Comment (vlach1989) [inline src/ReactLeafletMap/layers/CanvasVectorLayer/genericCanvasLayer.js:205]: why?

Comment (vdubr) [reply src/ReactLeafletMap/layers/CanvasVectorLayer/genericCanvasLayer.js:205]: I have error ```(!) Import of non-existent export``` in LeafletCanvasLayer.js without this export . Don`t you?

**Comment by vdubr** [comment]
Documentation published here: https://gisat.github.io/components/maps/presentational/reactLeaflet

**Comment by vdubr** [comment]
Documentation published here: https://gisat.github.io/components/maps/presentational/reactLeaflet

**Review by vlach1989 — APPROVED** [review]

**Pull request merged**